src/modules/esl-media/README.md

```diff
@@ -45,8 +45,16 @@ using a single tag as well as work with external providers using simple native-l
  - `ready-class` - class to add when the resource is ready
  - `ready-class-target` - [ESLTraversingQuery](../esl-traversing-query/README.md) to define a target for `ready-class`
 
- - `disabled` (boolean) - marker that prevents media api initialization
- 
+ - `disabled` (boolean) - marker that prevents media api initialization. Deprecated alias for *manual* mode of `lazy` attribute
```

Comment:
TODO: please check indents and correct rendering on the [GitHub](https://github.com/exadel-inc/esl/blob/esl-media-support-lazy-initialization/src/modules/esl-media/README.md) and first of all on the related site page